Ym search

marcinky opened the pull request:
I gave up on the fancy footwork and ended up generating a dupe search page. Renamed one to SearchFriend and the other to SearchHome. Updated App.js route witchcraft. Went back to links vice buttons on the landing.js page.

Commit:
Update Landing.js

Figuring out how to cause the search page to render different information depending on the button click of the landing page. (added buttons to landing page).

Commit:
Update Landing.js

Figuring out how to cause the search page to render different information depending on the button click of the landing page. (added buttons to landing page).

Commit:
Added search page

renamed both search pages, added to app.js (import), and updated landing.js to accomodate redirection. Wasted WAY too much time trying to be all fancy.

Commit:
Added search page

renamed both search pages, added to app.js (import), and updated landing.js to accomodate redirection. Wasted WAY too much time trying to be all fancy.

Commit:
removed buttons, went back to links

Links...keeping it simple for now.

Commit:
2 pulls behind after rebasing somehow

Commit:
Added some placeholder notes

Added placeholder notes

witherst commented on react-ui/src/App.js:
```diff
@@ -6,7 +6,8 @@ import Home from './components/home/Home';
 import Settings from './components/settings/Settings'
-import Search from './components/search/Search'
+import SearchFriend from './components/search/SearchFriend'
+import SearchHome from './components/search/SearchHome'
```
The only problem that I can see is that our search icon goes to the /search route right now. But the route is either /search/SearchFriend or /search/SearchHome.

When we click the Search icon, when should we go to SearchFriend and when should we go to SearchHome?  OR, which route should our Search Icon default to?

witherst replied:
Ans:  '/search/SearchFriend

Commit:
Update SearchFriend.js

witherst reviewed (approved):
Looks BUENNNNOOOOOOOOOOOOOO!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Commit:
Pictures MAN

I got "temp" pics to show using css and flex. (flex) (flex) (flex) (flex) (flex) (flex)
